python/cudf/cudf/core/column/column.py

```diff
@@ -528,6 +528,11 @@ def __setitem__(self, key: Any, value: Any):
         if out:
             self._mimic_inplace(out, inplace=True)
 
+    def _wrap_binop_normalization(self, other):
```

Comment:
Plead for a better name, even if it's longer. `_normalize_binop_value_with_NA`?

Reply:
I would like to punt on this and see if we can remove this method entirely in the future.